probe_src/probe_frontend/python/probe.py

```diff
@@ -0,0 +1,35 @@
+
+import typing
+import json
+import subprocess
+from . import ops
+
+OpTable = typing.Mapping[int, typing.Mapping[int, typing.Mapping[int, typing.List[ops.Op]]]]
+
+def load_log(path: str) -> OpTable:
+    ret: dict[int, dict[int, dict[int, list[ops.Op]]]] = {}
+
+
+    lines = subprocess.run(
+        ["probe", "dump", "--json", "--input", path], 
```

Comment:
Input should be a PROBE log not a PROBE record. The UX is that the user usually holds PROBE log and doesn't even notice that PROBE record got generated, transcribed, and deleted. Now, they want to analyze the PROBE log with Python.

Reply:
Yeah, that's how it works, maybe need to document it better, `probe dump --json` just dumps a probe log one op (with ID tripplet) per line with as json (this is documented in `cli/src/dump.rs` in the doc-comment for `to_stdout_json()`)

Reply:
Ah, I see